src/TextAnalysis.jl

```diff
@@ -50,6 +50,7 @@ module TextAnalysis
     export strip_numbers, strip_non_letters, strip_indefinite_articles, strip_definite_articles, strip_articles
     export strip_prepositions, strip_pronouns, strip_stopwords, strip_sparse_terms, strip_frequent_terms, strip_html_tags
     export SentimentAnalyzer
+    export PerceptronTagger, train, tag
```

Comment:
If you are exporting `train` and `tag`, then I guess it would be better to change the name to something hinting that it is specifically for the perceptron tagger. (Now we also have a Naive Bayes Classifier, soon more will come :slightly_smiling_face: )

Reply:
I will make them as `trainPerceptron` and `predictTag`. If you have any names please let me know

Reply:
Not sure, maybe not export it at all perhaps? Similar to Naive Bayes Classifier

Reply:
@laschuet May you suggest on this?

Reply:
> 
> 
> Not sure, maybe not export it at all, perhaps. Similar to Naive Bayes Classifier.

yes, even I am thinking of not exporting it

Reply:
Since, Naive Bayes Classifier is using `fit!` instead of `train` and `predict` instead of `tag` so I am thinking of using the same names for `Average Perceptron tagger` as well.